### PR TITLE
LPS-155671 [PRM] Search Page

### PR DESCRIPTION
**Pull request by ytuik**

### CODE REVIEW

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layout-page-templates/master-pages/main-template/page-definition.json [hunk 1]
```diff
@@ -2188,6 +2151,7 @@
 	},
 	"settings": {
 		"colorSchemeName": "01",
+		"css": "/*Source Serif Pro*/\n@font-face {\n\tfont-display: swap;\n\tfont-family: 'Source Sans Pro';\n\tfont-style: normal;\n\tfont-weight: 400;\n\tsrc: url(/documents/d/prm-01/sourcesanspro-regular-ttf);\n}",
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layout-page-templates/master-pages/main-template/page-definition.json:2154]: This setting is necessary? @ytuik @EuniceTravassos

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/08_search/page-definition.json [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+						"widthType": "Fluid"
+					}
+				},
+				"pageElements": [
+					{
+						"definition": {
+							"fragmentStyle": {
+								"backgroundFragmentImage": {
+									"title": {
+										"value": "banner_background_partner_portal.svg"
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/08_search/page-definition.json:30]: The file name need to be dash-case: banner-background-partner-portal.svg

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/08_search/page-definition.json [hunk 3]
```diff
@@ -0,0 +1,330 @@
...
+									},
+									"url": {
+										"mapping": {
+											"fieldKey": "FileEntry_previewImage",
+											"itemReference": {
+												"className": "com.liferay.portal.kernel.repository.model.FileEntry",
+												"classPK": "[$DOCUMENT_FILE_ENTRY_ID:/site-initializer/documents/group/banner_background_partner_portal.svg$]"
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/08_search/page-definition.json:37]:
```suggestion
												"classPK": "[$DOCUMENT_FILE_ENTRY_ID:/site-initializer/documents/group/banner-background-partner-portal.svg$]"
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/08_search/page-definition.json [hunk 4]
```diff
@@ -0,0 +1,330 @@
...
+										"actionKeys": [
+										],
+										"roleKey": "Site Member"
+									},
+									{
+										"actionKeys": [
+											"VIEW"
+										],
+										"roleKey": "Guest"
+									}
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/08_search/page-definition.json:312]:
```suggestion
									{
										"actionKeys": [],
										"roleKey": "Guest"
									}
```

Guest cannot have permissions

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layout-page-templates/master-pages/main-template/page-definition.json [hunk 5]
```diff
@@ -364,18 +341,8 @@
...
-																	"en_US": ""
-																}
-															},
 															"url": {
-																"mapping": {
-																	"fieldKey": "FileEntry_previewImage",
-																	"itemReference": {
-																		"className": "com.liferay.portal.kernel.repository.model.FileEntry",
-																		"classPK": "[$DOCUMENT_FILE_ENTRY_ID:/site-initializer/documents/group/placeholder/arrow-down-placeholder.png$]"
-																	}
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layout-page-templates/master-pages/main-template/page-definition.json:378]: This placeholder was removed, it is right? @ytuik @EuniceTravassos

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layout-page-templates/master-pages/main-template/page-definition.json [hunk 6]
```diff
@@ -264,47 +244,44 @@
...
-										"type": "Fragment"
+										"pageElements": [
+											{
+												"definition": {
+													"fragment": {
+														"key": "clay-icon-link"
+													},
+													"fragmentConfig": {
+														"iconLink": "./search",
+														"iconName": "search"
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layout-page-templates/master-pages/main-template/page-definition.json:275]: I think this doesn't works outside localhost. We need to mapping to page.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layout-page-templates/master-pages/main-template/page-definition.json [hunk 7]
```diff
@@ -80,29 +80,6 @@
...
-														},
-														{
-															"fieldName": "privatePage",
-															"fieldValue": "false"
-														},
-														{
-															"fieldName": "siteKey",
-															"fieldValue": "[$GROUP_KEY$]"
-														}
-													]
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layout-page-templates/master-pages/main-template/page-definition.json:101]: This placeholder was removed, it is right?

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layout-page-templates/master-pages/content-template/page-definition.json [hunk 8]
```diff
@@ -470,6 +468,7 @@
 	},
 	"settings": {
 		"colorSchemeName": "01",
+		"css": "/*Source Serif Pro*/\n@font-face {\n\tfont-display: swap;\n\tfont-family: 'Source Sans Pro';\n\tfont-style: normal;\n\tfont-weight: 400;\n\tsrc: url(/documents/d/prm-01/sourcesanspro-regular-ttf);\n}",
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layout-page-templates/master-pages/content-template/page-definition.json:471]: This setting is necessary?

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layout-page-templates/master-pages/content-template/page-definition.json [hunk 9]
```diff
@@ -264,47 +267,42 @@
...
+										"pageElements": [
+											{
+												"definition": {
+													"fragment": {
+														"key": "clay-icon-link"
+													},
+													"fragmentConfig": {
+														"iconLink": "./search",
+														"iconName": "search"
+													},
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layout-page-templates/master-pages/content-template/page-definition.json:297]: I think that we need to mapping a page

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

**Comment by jackelyneaquino** [comment]
jackelyneaquino

**Comment by EuniceTravassos** [comment]
Just starting reviewing :)

**Comment by jackelyneaquino** [comment]
jackelyneaquino

**Pull request closed without merging**